helm-charts/ibm-blockchain-network/templates/blockchain.yaml

```diff
@@ -28,18 +28,18 @@ spec:
     imagePullPolicy: {{.Values.blockchain.pullPolicy}}
     command: ["sh", "-c", "sleep 1 && while [ ! -f /shared/status_cryptogen_complete ]; do echo Waiting for cryptogen; sleep 1; done; echo 'sample config is =>' && ls /sampleconfig && cp /sampleconfig/configtx.yaml /shared/configtx.yaml; echo 'shared after copy is =>' && ls /shared && cd /shared/; configtxgen -profile TwoOrgsOrdererGenesis -outputBlock orderer.block && find /shared -type d | xargs chmod a+rx && find /shared -type f | xargs chmod a+r && touch /shared/status_configtxgen_complete && rm /shared/status_cryptogen_complete"]
     env:
-     - name: PEERHOST1
-       value: blockchain-org1peer1
-     - name: PEERPORT1
-       value: "30110"
-     - name: PEERHOST2
-       value: blockchain-org2peer1
-     - name: PEERPORT2
-       value: "30210"
-     - name: ORDERER_URL
-       value: blockchain-orderer:31010
-     - name: FABRIC_CFG_PATH
-       value: /shared
+    - name: PEERHOST1
```

Comment:
Can we use the same spacing for the file that existed before?

Reply:
the previous spacing was out and caused it to fail during the yaml to json conversion, that's why I changed it.
It works with the PR i submitted. 
I am on a Mac, not sure if that's a problem.